src/index.tsx

```diff
@@ -11,12 +11,6 @@ import { store } from 'src/store'
 
 BigNumber.set({ EXPONENTIAL_AT: [-7, 255] })
 
-if (process.env.NODE_ENV !== 'production') {
-  // eslint-disable-next-line
-  const whyDidYouRender = require('@welldone-software/why-did-you-render')
-  whyDidYouRender(React)
```

Comment:
We can use "React-components" plugins from devTools for this

Reply:
Not really, but I've not been using it in the past so ok